lib/ansible/modules/files/blockfromfile.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Evan Kaufman <evan@digitalflophouse.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: blockfromfile
+author:
+  - Evan Kaufman (@EvanK)
+version_added: "2.5"
+short_description: Search file from remote node using a provided regular expression
+description:
+  - This module will search a remote file for all instances of a pattern.
+  - Effectively the inverse of M(replace).
+options:
+  src:
+    required: true
+    aliases: [ path ]
+    description:
+      - The file to search.
+  regexp:
+    required: true
+    description:
+      - The regular expression to look for in the contents of the file.
+        Uses Python regular expressions; see
```

Comment:
It's probably better to make these sentences different items in the description.